pkg/odo/cli/url/create.go

```diff
@@ -241,12 +285,19 @@ func NewCmdURLCreate(name, fullName string) *cobra.Command {
 	}
 	urlCreateCmd.Flags().IntVarP(&o.urlPort, "port", "", -1, "port number for the url of the component, required in case of components which expose more than one service port")
 	urlCreateCmd.Flags().BoolVarP(&o.secureURL, "secure", "", false, "creates a secure https url")
-	// if experimental mode is enabled, add more flags to support ingress creation based on devfile
+	// if experimental mode is enabled, add more flags to support ingress creation or docker application based on devfile
 	if experimental.IsExperimentalModeEnabled() {
-		urlCreateCmd.Flags().StringVar(&o.tlsSecret, "tls-secret", "", "tls secret name for the url of the component if the user bring his own tls secret")
-		urlCreateCmd.Flags().StringVarP(&o.host, "host", "", "", "Cluster ip for this URL")
+		if pushtarget.IsPushTargetDocker() {
+			urlCreateCmd.Flags().IntVarP(&o.exposedPort, "exposed-port", "", -1, "an external port to the application container")
+			urlCreateCmd.Flags().BoolVarP(&o.forceFlag, "force", "f", false, "Don't ask for confirmation, assign an exposed port directly")
+			urlCreateCmd.Args = cobra.MaximumNArgs(0)
```

Comment:
Why can't we name the url? 
```
[mrinaldas@localhost nodejs-ex]$ odo url create example
Error: accepts at most 0 arg(s), received 1
Usage:
  odo url create [url name] [flags]
```
What am I missing? Please add a comment regarding that here.

Reply:
fixed.